src/Thinktecture.Blazor.ViewTransitions/ViewTransitionsServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Thinktecture.Blazor.ViewTransitions;
+
+public static class ViewTransitionsServiceCollectionExtensions
+{
+    /// <summary>
+    /// Adds all necessary services needed for the ViewTransition API
+    /// </summary>
+    /// <param name="services"></param>
+    public static void AddViewTransitionServices(this IServiceCollection services)
```

Comment:
```suggestion
    public static void AddViewTransitionService(this IServiceCollection services)
```

All the other service extension methods are singular too.

Reply:
Looks this is still plural.